lighthouse-core/computed/page-dependency-graph.js

```diff
@@ -463,6 +448,34 @@ class PageDependencyGraph {
     });
   }
 
+  /**
+   * Should provide the same urls found on `artifacts.URL`.
+   * TODO: make `artifacts.URL` a dependency for this computed artifact.
```

Comment:
I was hoping this would make it so we wouldn't need to refactor a bunch of test cases, but it seems like that's unavoidable since `networkRecordsToDevtoolsLog` doesn't currently add any `Page.frameNavigated` events.